action.yml

```diff
@@ -24,5 +28,30 @@ inputs:
     default: 'gpt-4-turbo-preview'
 
```

Comment:
🔒 Make sure to validate the environment variables before running the Python script to avoid runtime errors. You can add checks to ensure that required variables are set.